internal/tiltfile/tiltfile_state.go

```diff
@@ -395,6 +379,63 @@ func (s *tiltfileState) isWorkload(e k8s.K8sEntity) (bool, error) {
 	}
 }
 
+func (s *tiltfileState) applyK8sResourceOption(workload string, opts k8sResourceOptions) error {
+	sel, err := parseK8SObjectSelector(workload)
+	if err != nil {
+		return err
+	}
+
+	matched := false
+	var matchedRes *k8sResource
+	for _, r := range s.k8s {
+		if r.hasWorkloadEntity && sel.matches(r.workloadEntity) {
+			// ensure that we do not have multiple options per resource, or multiple resources per option
+			if matched {
+				return fmt.Errorf("matched two resources: %q and %q", matchedRes.name, r.name)
+			}
+			if r.k8sResourceOptionsApplied {
```

Comment:
what problem is this trying to solve? i can imagine cases where you'd want to override a previously set option. or if you want to set different options on different lines.